submissions/YuliiaDyka/oop-exercise/index.js

```diff
@@ -0,0 +1,60 @@
+class LivingBeing {
+   constructor(species, name, gender, saying) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.saying = saying;
+   };
+
+   showProperties() {
+      return Object.getOwnPropertyNames(this).map(prop => this[prop]).join('; ');
+   }
```

Comment:
Requirements re `Object` methods are still valid.

Also read about (and use) methods overloading/overriding and extended `super` API (calling inherited methods versions). You may want to revisit JS OOP educational materials.